### ISSUE TITLE
spawn EACCESS <3

### ISSUE THREAD

**Issue by samelie** [fluent-ffmpeg/node-fluent-ffmpeg#372]
I posted a question here, but there wasn't a fluent-ffmpeg tag so I'll ask here too.
http://stackoverflow.com/questions/28702999/error-spawn-eaccess-node-webkit-ffmpeg-with-fluent-ffmpeg

I want to bundle ffmpeg in with a nodewebkit app.

I downloaded the osx binaries from here: https://evermeet.cx/ffmpeg/ I'm assuming these are compiled but I could be wrong?

I get the fs.realpath to ffmpeg-2.5.4 and use that.
Using ./ffmpeg-2.5.4 or /ffmpeg-2.5.4 or ffmpeg-2.5.4 just gives a spawn ENOENT error which I've read, means not found.

If I remove setFfmepgPath from my fluent-ffmpeg command it works fine using my system ffmpeg.

**Comment by njoyard** [comment]
I think the current directory when running your app may not be what you'd expect. Can you please check what process.cwd() returns in your app (where you're calling fluent-ffmpeg) ?

**Comment by samelie** [comment]
`process.cwd()+'ffmpeg-2.5.4` **and** `fs.realpath('ffmpeg-2.5.4')` return the same path: '/private/var/folders/xm/01t0kntn0qxfs8m07ftghzq00000gp/T/.org.chromium.Chromium.l0asgaffmpeg-2.5.4<br>
however<br>The error message thrown by nodewebkit is different. (I changed the ffmpeg folder name to `ffmpeg` thinking this might solve it).
`process.cwd()+'ffmpeg'` throws the `spawn ENOENT` and the `realpath` throws `spawn EACCESS`.
I tried changing the permissions on the ffmpeg folder at runtime but that did nothing.
Got any ideas? Merci

**Comment by samelie** [comment]
Hey

I got it to work.
It probably helps if use the **build** version of ffpmeg... Turns out I decided **not** to use the binary from evermeet for whatever reason - they are the ones that work though! 
I put the executable binary in the root.
The following works when the app is `.nw` and when packaged into a `.app`.
When it's time to call fluent-ffmpeg:<br>I get the path to ffmpeg binary: `process.cwd()+'/ffmpeg'`<br>I then do a `fs.chmod(pp, '777', function()`, inside the callback I do the fluent-ffmpeg command and `setFfmpegPath` to that same path
Yey!
